tests/scripts/sai_qualify/DUTScript/syncd_rpc_shell/init_env.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+/usr/bin/start.sh
+. /usr/bin/syncd_init_common.sh
+config_syncd
```

Comment:
Is there a strong justification for adding this top-level folder in repository?
Can this file be moved to some other subfolder to avoid adding a top-level folder?

Reply:
thanks for the suggestion.
add more comments in that shell.

the commands that been used within this shall are only available within syncd or syncd-rpc docker.
this shell can only be available within syncd-rpc and syncd docker. it will be used to control syncd-rpc start process for debugging.